src/plugins/data_source_management/common/data_sources/types.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ToastInputFields, ErrorToastOptions } from 'src/core/public/notifications';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import type { SavedObject } from 'src/core/server';
+
+export interface IDataSource {
+  title: string;
+  id?: string;
+  type?: string;
+  endpoint: string;
+}
+
+export interface IDataSourceAttributes {
+  title: string;
+  id: string;
+  type: string;
+  endpoint: string;
+}
+
+export type OnNotification = (toastInputFields: ToastInputFields) => void;
+export type OnError = (error: Error, toastInputFields: ErrorToastOptions) => void;
+
+export interface UiSettingsCommon {
+  get: (key: string) => Promise<any>;
+  getAll: () => Promise<Record<string, any>>;
+  set: (keu: string, value: any) => Promise<void>;
+  remove: (key: string) => Promise<void>;
+}
+
+export interface SavedObjectsClientCommonFindArgs {
```

Comment:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/76021805ac38e7a212838908359951c0032a1c68/src/plugins/data/common/index_patterns/types.ts#L85-L108
Same issue here.

Reply:
removed this file